src/Makefile

```diff
@@ -22,6 +22,9 @@ install_deb = \
 libnl-%.deb:
 	pushd libnl3; ./build.sh; popd
 
+libteam5_1.26-1_amd64.deb libteam-dev_1.26-1_amd64.deb libteam-utils_1.26-1_amd64.deb libteamdctl0_1.26-1_amd64.deb:
```

Comment:
need to add LIBNL-DEBS here, otherwise because building them requires libnl, I believe.